stack.yml

```diff
@@ -133,6 +145,8 @@ resources:
                   curl https://raw.githubusercontent.com/osism/testbed/master/playbooks/manager-part-1.yml | sudo -iu dragon tee /home/dragon/manager-part-1.yml
                   curl https://raw.githubusercontent.com/osism/testbed/master/playbooks/manager-part-2.yml | sudo -iu dragon tee /home/dragon/manager-part-2.yml
                   sudo -iu dragon ansible-playbook -i localhost, /home/dragon/manager-part-1.yml
+                  sudo -iu dragon sh -c 'cd /opt/configuration; ./scripts/set-ceph-version.sh ceph_version'
```

Comment:
Does this work this way? I think a str_replace call is missing here?

https://docs.openstack.org/heat/latest/template_guide/software_deployment.html